app/controllers/users_controller.rb

```diff
@@ -1,26 +1,33 @@
 # frozen_string_literal: true
 
 class UsersController < ApplicationController
-  def index; end
+  def index
+    @users = User.all
+  end
 
   def show
     @user = User.find_by(id: params[:id])
-    render json: @user
+    return @user if @user
+
+    redirect_to root_path
   end
 
+  def new; end
+
   def create
     @user = User.create(user_params)
     if @user.save
       flash[:success] = t("create.success_message", name: t("create.label_name"))
+      redirect_to @user
     else
       flash[:errors] = @user.errors.full_messages
```

Comment:
Flash này ko hợp lý, nếu em đưa value của flash là các mess invalid thì dẫn đến content của flash rất nhiều

Reply:
nên flash đơn giản như create fail
và 1 màn render partial để show các mess invalid for field vào